### DISCUSSION TITLE
Correct Font configuration to use "IntelOne Mono"

### DISCUSSION THREAD

**Discussion by TuxxuT** [kovidgoyal/kitty#6349] [General]
Hello,

I want to use the IntelOne Mono font as my standard font in Kitty under Arch Linux.
https://github.com/intel/intel-one-mono
I installed the font via AUR (yay).

kitty +list-fonts listed the font successfully as:
    IntelOne Mono
    IntelOne Mono
    IntelOne Mono Bold
    IntelOne Mono Bold Italic
    IntelOne Mono Italic
    IntelOne Mono Light
    IntelOne Mono Light Italic
    IntelOne Mono Medium
    IntelOne Mono Medium Italic

The question is: What do I have to write in the kitty.conf to use the font?
The line with the font_family option look currently like: "font_family      intelone jetbrains mono nerd font"

But this does not work...

**Comment by kovidgoyal** [comment]
font_family IntelOne Mono

**Comment by TuxxuT** [comment]
Thanx, I tried it out -> without success. :-(

**Comment by kovidgoyal** [comment]
Use --debug-font-fallback
